src/tests/test_x509_tn_auth_list.cpp

```diff
@@ -0,0 +1,80 @@
+/*
+* (C) 2006,2011,2012,2014,2015 Jack Lloyd
+* (C) 2022 René Meusel, Rohde & Schwarz Cybersecurity
```

Comment:
Copy-pasta. :) ... Feel free to add your credits in the files you modify (significantly).